src/chrome/content/rules/AmamiyaSora.jp.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="AmamiyaSora.jp">
+	<target host="amamiyasora.jp" />
+	<target host="www.amamiyasora.jp" />
+
+	<target host="^http:" to="https:" />
```

Comment:
Travis is failing for this PR.  it should be `rule from` instead of `target host` here.

Reply:
Sorry for stupid typo, will fix.